packages/gluestick/src/renderer/components/Body.js

```diff
@@ -9,48 +9,29 @@ export default class Body extends Component {
     isEmail: PropTypes.bool.isRequired,
     initialState: PropTypes.any.isRequired,
     envVariables: PropTypes.array.isRequired,
-    scriptTags: PropTypes.array.isRequired,
+    scriptTags: PropTypes.string.isRequired,
   };
 
   render() {
-    const { isEmail } = this.props;
```

Comment:
Reduced method sprawl, it was difficult to figure out what was going where in here.